tsdb/mapper.go

```diff
@@ -181,6 +182,11 @@ func (lm *LocalMapper) Open() error {
 		selectTags.add(tsf.selectTags...)
 		whereFields.add(tsf.whereFields...)
 
+		// If we only have tags in our select clause we just return
+		if len(selectFields) == 0 && len(selectTags) > 0 {
```

Comment:
Yeah, I think it should be an error, but these cases are no longer simple due to the distributed nature of the system. I would be OK with opening a 0.9.4 ticket, and coming back to this.